homeassistant/components/hassio/strings.json

```diff
@@ -9,7 +9,7 @@
       "host_os": "Host Operating System",
       "installed_addons": "Installed Add-ons",
       "supervisor_api": "Supervisor API",
-      "supervisor_version": "Supervisor Version",
+      "supervisor_version": "Version",
```

Comment:
I think there is no harm in leaving it here (even though as a duplicate, double clarity?).
For example, the one above is also `supervisor_api` -> `Supervisor API`.

Reply:
true